torchvision/prototype/transforms/_transform.py

```diff
@@ -6,9 +6,18 @@
 from torch import nn
 from torch.utils._pytree import tree_flatten, tree_unflatten
 from torchvision.prototype.features import _Feature
+from torchvision.prototype.transforms._utils import is_simple_tensor
 from torchvision.utils import _log_api_usage_once
 
 
+def _isinstance(obj: Any, types: Tuple[Type, ...]) -> bool:
+    has_tensor = torch.Tensor in types
+    if not has_tensor:
+        return isinstance(obj, types)
+    types_ = tuple(t for t in types if t != torch.Tensor)
+    return isinstance(obj, types_) or is_simple_tensor(obj)
```

Comment:
In other places, we are very careful on when we use `torch.Tensor` and when we use `is_simple_tensor()`. Here handle it differently. Why not use a similar approach as for `has_any` and `has_all` and accept a `*types_or_checks: Union[Type, Callable[[Any], bool]]`?